dashboard/common/db_util.py

```diff
@@ -92,6 +93,14 @@ class CourseEnrollment(TypedDict):
     course_name: str
     enrollment_types: List[str]
 
+def is_superuser(user_name: str) -> bool:
+    logger.info(is_superuser.__name__+f' \'{user_name}\'')
```

Comment:
If we want to log this, maybe it is better to also log the result of is_superuser call. Like after line 103

I think the log level could be debug instead of info.